CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Change Log
 
+## 2.13.0 - 2022-11-08
+- Oversampling Mitigation
+[PR356](https://github.com/aws/aws-xray-sdk-java/pull/356)
+- Add back original subsegment noOp method
+[PR359](https://github.com/aws/aws-xray-sdk-java/pull/359)
+- Refactoring TraceHeader Calls
```

Comment:
Change `PR359` to `PR 359`
Also I think we can remove the "Refactoring" items from from the Changelog. Usually this is not included.